### PR TITLE
Fix indent when after comment

### PR DESCRIPTION
**Pull request by uk-ar**
I fixed indent after comment.
- expected result:
```swift
func foo() {
    foo()
    //
    |foo()
}
```
- actual:
```swift
func foo() {
    foo()
    //
        |foo()
}
```

### CODE REVIEW

File: swift-mode.el [hunk 1]
```diff
@@ -225,7 +225,10 @@
              (looking-at "[ \n\t]+[.?:]")
              ;; Operators placed on the second line in multi-line expression
              ;; Should respect here possible comments strict before the linebreak
-             (looking-at (concat "\\(\/\/.*\\)?\n[[:space:]]*" swift-smie--operators-regexp))
+             (and (looking-at
+                   (concat "\\(\/\/.*\\)?\n[[:space:]]*" swift-smie--operators-regexp))
+                  (not (looking-at
+                        (concat "\\(\/\/.*\\)?\n[[:space:]]*" "//"))))
```
Comment (ap4y) [inline swift-mode.el]: I think string concatenation here is unnecessary. Also I think `not` branch here can be probably included into positive lookup regex. What do you think?

**Comment by uk-ar** [comment]
I agree with you :smile: . I will try.

**Comment by uk-ar** [comment]
I improved this patch as you comment. But, Travis CI seems to fail with another reason. :cry:

**Comment by ap4y** [comment]
Thanks for the changes @uk-ar. This fix with regex looks ok, but I don't like semantics of adding `comment` regex into `swift-smie--operators-regexp` which holds regex for operators. Maybe the better solution is to create new implicit semicolon rule [here](https://github.com/uk-ar/swift-mode/blob/fix-indent-after-comment/swift-mode.el#L222) with comment? Also may I ask you to squash commits into one? Thanks a lot!

**Comment by uk-ar** [comment]
I fixed broken implicit semicolon rule. But I think it is not good at squash commits, because as far as I know squash commits also delete diff comment. How do you think?

**Comment by ap4y** [comment]
You can use different strategies during rebase as shown [here](http://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Changing-Multiple-Commit-Messages). This will let you merge comments, leave one comment or even change it. Squashing commits is a normal practice for keeping main branch clean. For this PR it make sense, having 3 commits per 1 bugfix is not ideal. Hope it makes sense :)

**Comment by uk-ar** [comment]
OK, I understand :smile: . I integrate last 2 commits to 1 commit, And leave 1st commit for logging discussion. How about this?

**Comment by ap4y** [comment]
That would be awesome! :)

**Comment by ap4y** [comment]
Hey @uk-ar did I miss update here by any chance?

**Comment by uk-ar** [comment]
Hi, @ap4y. I had intended to squash already, 3 commits to 2 commit. Could you please review it?

**Comment by ap4y** [comment]
I will merge it because I don't want to bother you again, but would be ideal to squash a small related changes into one commit. Thanks!

**Pull request merged**